### PR TITLE
fix(PopupMenu): show `emptyPlaceholder` when no entries exist

### PR DESCRIPTION
**Pull request by marstamm**
This allows us to show empty placeholder when no entries were returned (e.g. still fetching), not just when the search filters out all entries

### CODE REVIEW

**Review by nikku — COMMENTED** [review]
If I understand this unlocks the following use-case:

* Regardless of whether search is active or not, if no entries exist then the `emptyPlaceholder` is shown.

Is that correct?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ All notable changes to [diagram-js](https://github.com/bpmn-io/diagram-js) are d
 
 _**Note:** Yet to be released changes appear here._
```
Comment (nikku) [inline CHANGELOG.md:8]: The component is `popup-menu`, please adjust the commit accordingly.

**Comment by nikku** [comment]
Fine to merge then with `popup-menu` as the touched upon component. I'll cut a release soon and will include this change.

**Comment by marstamm** [comment]
Already adjusted the commit message

**Pull request merged**